src/commands/version.rs

```diff
@@ -1,6 +1,6 @@
 //! The `version` subcommand
 //!
-#![allow(unknown_lints, never_loop)]
+#![allow(unknown_lints, clippy::never_loop)]
```

Comment:
Maybe you can get rid of `unknown_lints` now?

Reply:
removed `unknown_lints` and looks like clippy approved.